openapi.yaml

```diff
@@ -5290,7 +5300,7 @@
           "teamname" : {
             "type" : "string"
           },
-          "cluster" : {
+          "envName" : {
             "type" : "string"
           },
```

Comment:
I think you just need to run the pnpm creation command here to update the types

Reply:
src/domain/topic/topic-test-helper.ts:18:5 - error 
src/domain/topic/topic-test-helper.ts:50:5 - error 

@mathieu-anderson

Reply:
I'm a little confused at seeing the generated types: `envName` is mandatory but `envId` is not? 

Also, for clarification: `envName` would be `"DEV"` and `envId` would be `"1"` right?

Reply:
yes envName would be "DEV" and envId would be "1"., but envId will be mandatory. Updated.